csr/csr_to_socrata.py

```diff
@@ -63,23 +63,14 @@ def extract():
     logger.info(f"Downloaded {len(df)} CSRs from endpoint")
     return df
 
-
 def convert_from_state_plane(df):
     """
-    Adds WGS-84 lat/long columns to the dataframe based on the state plane coordinates.
+    Adds a WGS-84 lat/long column to the dataframe based on the state plane coordinates.
     """
-    gdf = gpd.GeoDataFrame(
-        df,
-        geometry=gpd.points_from_xy(
-            x=df["State Plane X Coordinate"], y=df["State Plane Y Coordinate"]
-        ),
-        crs="ESRI:102739",
-    )
-    gdf = gdf.to_crs("EPSG:4326")
-
-    # Get wgs84 location columns
-    df["latitude"] = gdf["geometry"].y
-    df["longitude"] = gdf["geometry"].x
+    # projection of coordinates
+    transformer = Transformer.from_crs(crs_from="ESRI:102739", crs_to="EPSG:4326")
```

Comment:
Thanks for the @ @johnclary! 

I couldn't recall having seen SRID 102739 anytime recently. I looked it up, and it's a deprecated version of our Central State Plan projection that works great for us here in ATX and is used a lot in the department. I pulled up the proj4 string for it and compared it to 2277, and they are the same for all intents and purposes - so it should just work for you. If we have that data somewhere under our control that you're transforming from, we might want to update its metadata by reprojecting it into 2277. Not a big deal, but interesting at least. I'd be a house-keeping thing.